openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/TestLibcImplementation.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * The copyright of this file belongs to Feedzai. The file cannot be
+ * reproduced in whole or in part, stored in a retrieval system,
+ * transmitted in any form, or by any means electronic, mechanical,
+ * photocopying, or otherwise, without the prior permission of the owner.
+ *
+ * © 2023 Feedzai, Strictly Confidential
+ */
+package com.feedzai.openml.provider.lightgbm;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+/**
+ * Tests the retrieval of libc implementation.
+ *
+ * @author Renato Azevedo (renato.azevedo@feedzai.com)
+ */
+public class TestLibcImplementation {
+    @Test
+    public void unknownLibcImplementationThrowsException() {
+        Assertions.assertThatThrownBy(() -> LightGBMUtils.getCpuArchitecture("klibc"))
```

Comment:
I think you are testing the wrong method

Reply:
Thanks! I fixed it in the latest commits.